src/core/components/sidepanels/panels/theme-configuration/BorderRadiusInput.tsx

```diff
@@ -1,15 +1,22 @@
 import { debounce } from "lodash-es";
 
-const BorderRadiusInput = ({ onChange, disabled }: { onChange: (value: string) => void; disabled: boolean }) => {
+type BorderRadiusInputProps = {
+  value: string;
+  onChange: (value: string) => void;
+  disabled?: boolean;
+};
+
+const BorderRadiusInput = ({ value, onChange, disabled }: BorderRadiusInputProps) => {
   const throttledChange = debounce((value: string) => onChange(value), 200);
+
   return (
     <input
       type="range"
       min="0"
       step="1"
       max="30"
       disabled={disabled}
-      defaultValue="15"
+      defaultValue={value.replace("px", "")}
```

Comment:
Will is save as `px` at the time of save?

Reply:
We only need this for once . Yeah it will save as px